tests/test_integrity.py

```diff
@@ -75,14 +77,32 @@ def test_references(db):
     # Confirm that all are in Publications
     t = db.query(db.Publications.c.publication).filter(db.Publications.c.publication.in_(ref_list)).astropy()
     assert len(t) == len(ref_list), 'Some references were not matched'
+    assert len(t) == 578, f'{len(t)} publications referenced'
```

Comment:
Let's think about this. Do we really want to modify this test every time we add a reference?

Reply:
No, lets get rid of it.